Formula/wso2ei.rb

```diff
@@ -0,0 +1,43 @@
+class Wso2ei < Formula
+  desc "WSO2 Enterprise Integrator"
+  homepage "https://wso2.com/integration/"
+  url "https://dl.bintray.com/wso2/binary/wso2ei-6.4.0.zip"
+  sha256 "36ab3982cc517674ab1673c1947093bf8022e1f4fb971a2fe6fa1b3085a13917"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8"
+
+  def install
+    %w[
+      integrator
+      business-process
+      broker
+      analytics-dashboard
+      analytics-worker
+      micro-integrator
+      msf4j
+    ].each do |profile|
+      bin.install "bin/wso2ei-#{version}-#{profile}" => "wso2ei-#{profile}"
+    end
+
+    libexec.install Dir["*"]
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.8"))
+  end
+
+  test do
+    ENV["CARBON_HOME"] = testpath
+    cp_r Dir["#{libexec}/*"], testpath
```

Comment:
We shouldn't be copying the entire installation into the testpath. It should run from where it is installed.

Reply:
We have a challenge when running tests against the install directory as it depends on OpenJDK we ship with our package. We followed the same practice followed by Apache Tomcat brew package (https://github.com/Homebrew/homebrew-core/blob/master/Formula/tomcat.rb).

If Tomcat way is wrong, could you please suggest a better way to run the test against a different JDK?

Thanks!

Reply:
We are making the announcement of choosing Homebrew as the package distributor for WSO2 runtimes. We really want to host these packages ASAP before the pubic announce is made. I really apreciate your support on this.

Thanks!